sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/models/properties/TypedPropertyUtils.java

```diff
@@ -0,0 +1,47 @@
+package com.microsoft.appcenter.ingestion.models.properties;
+
+import android.support.annotation.NonNull;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.microsoft.appcenter.ingestion.models.CommonProperties.TYPE;
+import static com.microsoft.appcenter.ingestion.models.CommonProperties.TYPED_PROPERTIES;
+
+public class TypedPropertyUtils {
+
+    @SuppressWarnings("IfCanBeSwitch")
```

Comment:
Why not use a switch, then?

Reply:
Because of https://stackoverflow.com/a/42680333/598520.
And we cannot update Jacoco version until Android plugin itself uses a more recent version (3.2 is out and we have a task to update to 3.2 so we might be able to, but not guaranteed).